src/core/queryClient.ts

```diff
@@ -123,6 +123,31 @@ export class QueryClient {
       .setData(updater, options)
   }
 
+  setQueriesData<TData>(
+    queryKey: QueryKey,
+    updater: Updater<TData | undefined, TData>,
+    options?: SetDataOptions
+  ): [QueryKey, TData][]
+
+  setQueriesData<TData>(
+    filters: QueryFilters,
+    updater: Updater<TData | undefined, TData>,
+    options?: SetDataOptions
+  ): [QueryKey, TData][]
```

Comment:
not sure if this is the best return type - wanted to provide a way to show which key actually had data set, as calling `setQueriesData` on a filter that doesn't match anything will not set anything